app/code/Magento/InstantPurchase/Model/ShippingMethodChoose/DeferredShippingMethodChooserPool.php

```diff
@@ -15,6 +15,9 @@ class DeferredShippingMethodChooserPool
 {
     private $choosers;
 
+    /**
+     * @param array $choosers
```

Comment:
Please be more specific about type, `DeferredShippingMethodChooserInterface[]` can be used here instead of generic array.